recipes/tike/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "tike" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tomography/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 89aafa54a04d1ca58ab121958bdcbd01cfd521ecfd70312fbe174a909009faa3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py<35 or (not unix)]
+
+requirements:
```

Do we need `git` here too?

```suggestion
requirements:
  build:
    - git
```

---

If it's needed it should implicit from `setuptools_scm_git_archive` rather than being explicitly listed.